docs/how-to-guides/monitoring-with-prometheus-operator.md

```diff
@@ -1,4 +1,4 @@
-# Setting up monitoring on Lokomotive
+# Set up monitoring on Lokomotive
 
 ## Contents
```

Comment:
Contents structure differs from other how-to-guides.

Reply:
Do you mean like steps?

Reply:
For prometheus operator there are very less steps. And it has other things like small changes which reflect in other places, I am not sure if steps add any value here? WDYT?

Reply:
Still it would give a consistent feel to the docs.

Reply:
done.